### PR TITLE
Update keycloak container image

### PR DESCRIPTION
**Pull request by basisbit**
Keycloak 16.x is out of support. The current version is 19.0.2. The maintainer moved to quay.io for container hosting, and container image version 18 had some bigger changes.

Made these changes on request, but I don't have any infrastructure or keycloak setup to test this myself.
Can someone test this and provide feedback? Thanks!

### CODE REVIEW

**Comment by KH-Amir-TN** [comment]
Hi @basisbit,

Thanks for your contribution!
Indeed, the current used image of **Keycloak** isn't aligned with the latest official releases.
We've been already planning to change the image in use but since you already did that we'll be happy to have your PR.
However, this is still a draft.
If you still plan to make some changes then please go ahead and then mark this PR as **ready for review**.
If you believe that this PR is ready then kindly update its state.

We'll further test your PR once **ready**.

Amir,

**Comment by basisbit** [comment]
@KH-Amir-TN please see my comment above. I was waiting for someone who has a working keycloak setup to test if this change works.

File: docker-compose.yml [hunk 1]
```diff
@@ -69,24 +69,27 @@ services:
...
+      KC_HOSTNAME_STRICT: false
+      KC_EDGE: proxy
     volumes:
       - ./data/certbot/conf/:/etc/letsencrypt
       - ./data/certbot/conf/live/kc.${DOMAIN_NAME:-xlab.blindside-dev.com}/cert.pem:/etc/x509/https/tls.crt
       - ./data/certbot/conf/live/kc.${DOMAIN_NAME:-xlab.blindside-dev.com}/privkey.pem:/etc/x509/https/tls.key
     depends_on:
       - postgres
+    command:
+      - start-dev --auto-build
```
Comment (jfederico) [inline docker-compose.yml:92]: It would have to run in production mode "start"

**Comment by jfederico** [comment]
Thanks for the PR @basisbit but if you cannot even test it you should not open an hypothetical PR. Perhaps open an issue as a requested enhancement.

**Pull request closed without merging**